src/styles/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin rotate-animation($duration, $timing-function) {
+  animation: hand-move $duration $timing-function infinite;
+  transform-origin: bottom center;
+
+  @keyframes hand-move {
+    from {
+      transform: rotate(0deg);
+    }
+    to {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
The `@keyframes` definition should be placed globally and not inside a mixin to avoid duplicating the keyframes when the mixin is included in multiple places. Define `@keyframes hand-move` outside of the mixin.